### PR TITLE
feat(sbb-image): provide load and error events

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/elements/image/image.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    expect(errorSpy).not.to.have.been.called;
+  });
+
+  it('should trigger error event', async () => {
+    const loadSpy = spy();
+    const errorSpy = spy();
+
+    await fixture(
+      html`<sbb-image
+        image-src="http://localhost/er.png"
```
Comment (jeripeierSBB) [inline src/elements/image/image.e2e.ts:37]:
```suggestion
        image-src="http://localhost/dummy.png"
```

File: src/elements/lead-container/lead-container.snapshot.spec.ts [hunk 2]
```diff
@@ -11,19 +13,22 @@ import '../link/block-link/block-link.js';
...
   let element: SbbLeadContainerElement;
 
   beforeEach(async () => {
     element = await fixture(
       html`<sbb-lead-container>
-        <sbb-image slot="image"></sbb-image>
+        <sbb-image slot="image" image-src=${imageUrl}></sbb-image>
       </sbb-lead-container>`,
     );
+    await waitForImageReady(element.querySelector<SbbImageElement>('sbb-image')!);
```
Comment (kyubisation) [inline src/elements/lead-container/lead-container.snapshot.spec.ts:27]: Is the generic Parameter needed? It should default to `SbbImageElement` from my understanding.

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Thanks!

File: src/elements/image/image.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { spy } from 'sinon';
+
+import { fixture } from '../core/testing/private.js';
+import { waitForCondition } from '../core/testing.js';
+
+import './image.js';
+
+const imageUrl = import.meta.resolve('../clock/assets/sbb_clock_face.svg');
+
+describe(`sbb-image`, () => {
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/elements/image/image.e2e.ts:12]: I want to make sure it's going through the load event and not the image.complete. For every other test in all the project I'd also recommend to use the utility function :).

**Pull request merged**